serial_server.py

```diff
@@ -216,13 +217,13 @@ def baudrate(self, c, data=None):
     def bytesize(self, c, data=None):
         """Sets the bytesize."""
         ser = self.getPort(c)
-        bsizes = [long(x[1]) for x in ser.getSupportedByteSizes()]
+        bsizes = [long(x[1]) for x in ser.BYTESIZES]
```

Comment:
This should just be bsizes = ser.BYTESIZES (or just get rid of the temporary altogether)